# Álgebra para analizar objetos geométricos

# Clase 4

### Paralelismo y perpendicularidad

Pensemos que tenemos dos rectas:

* Primera recta, $y=m_1x+b_1$
* Segunda recta, $y=m_2x+b_2$

Estas rectas son paralelas si

$$ m_1=m_2 $$

Estas rectas son perpendiculares si

$$ m_1m_2=-1 $$

* ***Ejemplo*** Tenemos la ecuación de la recta $y=6x-2$. Construye otra recta que sea perpendicular.

* Datos:
    * $m_1=6$
    * $b_1=-2$
    * $m_2=?$
    * $b_2=?$

* Fórmula:

$$ m_1m_2=-1 $$

$$ m_2=\dfrac{-1}{m_1} $$

* Sustitución:

$$ m_2=-\dfrac{1}{6} $$

*(Básicamente, si queremos la pendiente perpendicular basta con que volteemos la fracción de nuestra pendiente y su signo.)*

Entonces, cualquier recta con pendiente $-\dfrac{1}{6}$ es perpendicular a la recta $y=6x-2$, independientemente de la ordenada al origen que usemos para la segunda recta.

P.ej.:

$$ y=-\dfrac{1}{6}x-5 $$

<img src="perp.png" align="middle" width="400">

* ***Ejemplo*** Tenemos la ecuación de la recta $y=\dfrac{3}{2}x-2$. Construye otra recta que sea perpendicular.

$$ y=-\dfrac{2}{3}x+10 $$

### Ángulo entre dos rectas conociendo sus pendientes

Para encontrar el ángulo entre dos rectas:

* Primera recta, $y=m_1x+b_1$
* Segunda recta, $y=m_2x+b_2$

utilizamos la siguiente fórmula

$$ \theta =\tan^{-1}\left(\dfrac{m_2-m_1}{1+m_2m_1}\right) $$

* ***Ejemplo*** Tenemos dos rectas:
    * La primera recta es $x+2y-1=0$
    * La segunda recta es $9x-3y+21=0$
    
    ¿Cuál es el ángulo que se forma entre mis dos rectas?

Forma general:
$$ Ax+By+C=0 $$
Forma ordenada-pendiente:
$$y=mx+b$$

Primero las pasamos a su forma general:

* Para la primera recta:

$$x+2y-1=0$$

$$2y=-x+1$$

$$ y=\dfrac{-x+1}{2} $$

$$ y=\dfrac{-x}{2}+\dfrac{1}{2} $$

$m=-\dfrac{1}{2}$

* Para la segunda recta

$$9x-3y+21=0$$

$$-3y=-9x-21$$

$$y=\dfrac{-9x-21}{-3}$$

$$y=\dfrac{-9x-21}{-3}$$

$$y=\dfrac{-9x}{-3}+\dfrac{-21}{-3}$$

$$y=3x+7$$

$m=3$

$$ \theta =\tan^{-1}\left(\dfrac{m_2-m_1}{1+m_2m_1}\right) $$

In [1]:
2+2

4

In [2]:
atand((3-(-1/2))/(1+(3)(-1/2)))

-81.86989764584403

In [3]:
atand(((1/3)-(-4/5))/(1+(1/3)*(-4/5)))

57.0947570770121

* ***Ejemplo*** Tenemos los vértices de un triángulo dado por las coordenadas $A(7,-3)$, $B(-11,4)$, y $C(3/2,5)$. ¿Cuáles son los ángulos internos del triángulo?

* La pendiente del segmento $\overline{AB}$ 

$$ m=\dfrac{y_2-y_1}{x_2-x_1} $$

$$ m=\dfrac{(4)-(-3)}{(-11)-(7)} $$

$$ m=\dfrac{7}{-18}=-\dfrac{7}{18} $$

* La pendiente del segmento $\overline{BC}$ 

$$ m=\dfrac{y_2-y_1}{x_2-x_1} $$

$$ m=\dfrac{(4)-(5)}{(-11)-(1.5)} $$

$$ m=\dfrac{-1}{-12.5}=\dfrac{2}{25} $$

* La pendiente del segmento $\overline{AC}$

$$ m=\dfrac{y_2-y_1}{x_2-x_1} $$

$$ m=\dfrac{(5)-(-3)}{(1.5)-(7)} $$

$$ m=\dfrac{8}{-5.5}=-\dfrac{16}{11} $$

* El ángulo que se forma entre los segmentos $\overline{AB}$ y $\overline{AC}$ ($\angle BAC$)

$$ \theta =\tan^{-1}\left(\dfrac{m_2-m_1}{1+m_2m_1}\right) $$

In [4]:
atand(((-7/18)-(-16/11))/(1+(-7/18)*(-16/11)))

34.24097150519836

* El ángulo que se forma entre los segmentos $\overline{AB}$ y $\overline{BC}$ ($\angle ABC$)

$$ \theta =\tan^{-1}\left(\dfrac{m_2-m_1}{1+m_2m_1}\right) $$

In [5]:
atand(((-7/18)-(2/25))/(1+(-7/18)*(2/25)))

-25.8244267670341

* El ángulo que se forma entre los segmentos $\overline{AC}$ y $\overline{BC}$ ($\angle ABC$)

$$ \theta =\tan^{-1}\left(\dfrac{m_2-m_1}{1+m_2m_1}\right) $$

In [6]:
atand(((2/25)-(-16/11))/(1+(2/25)*(-16/11)))

60.065398272232464

In [7]:
180-60.065398272232464

119.93460172776753

* $\angle BAC=34.24^\circ$

* $\angle ABC=25.82^\circ$

* $\angle ACB=119.93^\circ$

La suma de los ángulo internos de un triángulo es $180^\circ$. Compramos esto:

In [8]:
34.24097150519836+25.8244267670341+119.93460172776753

180.0

### Distancia de un punto a una recta

Supongamos que tenemos una recta en su forma general
$$ Ax+By+C=0 $$

Pensemos que tenemos un punto $P(x_0,y_0)$

La distancia entre el punto y la recta es

$$ d=\dfrac{|Ax_0+By_0+C|}{\sqrt{A^2+B^2}} $$

**Reglas:**
* Si me dan la ecuación de la recta en la forma ordenada-pendiente tengo que pasarla a su forma general. Si me la dan en la forma general vamos al siguiente paso
* Señalamos cuánto valen los coeficientes de la recta en su forma general
* Sustituimos las coordenadas del punto $P(x_0,y_0)$ en la forma general de la ecuación de la recta y evaluamos
* Las línes verticales en la fórmula significa *valor absoluto*, es decir, el valor que obtengamos lo mantemos positivo.

* ***Ejemplo:*** Tenemos la recta $y=x+2$ y el punto $P(1,1)$. ¿Cuál es la distancia entre ellos?

$$ y=x+2 $$

$$ -x+y-2=0 $$

$A=-1$, $B=1$, $C=-2$

$$ d=\dfrac{|Ax_0+By_0+C|}{\sqrt{A^2+B^2}} $$

$$ d=\dfrac{|(-1)(1)+(1)(1)+(-2)|}{\sqrt{(-1)^2+(1)^2}} $$

$$ d=\dfrac{|-1+1-2|}{\sqrt{1+1}} $$

$$ d=\dfrac{|-2|}{\sqrt{2}} $$

$$ d=\dfrac{2}{\sqrt{2}} $$

In [9]:
2/(√2)

1.414213562373095

¿Cuánto es la distancia al punto $Q(2,4)$?

$A=-1$, $B=1$, $C=-2$

$$ d=\dfrac{|Ax_0+By_0+C|}{\sqrt{A^2+B^2}} $$

$$ d=\dfrac{|(-1)(2)+(1)(4)+(-2)|}{\sqrt{(-1)^2+(1)^2}} $$

$$ d=\dfrac{|-2+4-2|}{\sqrt{1+1}} $$

$$ d=\dfrac{0}{\sqrt{2}}=0 $$

*"Cualquier punto que se encuentre sobre mi recta, tiene una distancia cero a esa recta."*

<img src="dist_punto_recta.png" align="middle" width="400">